Solutions/Snowflake/Analytic Rules/SnowflakeUserAddAdminPrivileges.yaml

```diff
@@ -0,0 +1,33 @@
+id: f258fa0c-e26c-4e2b-94fb-88b6cef0ca6e

```

Comment:
There should not be 2 templates with the same ID, but the id f258fa0c-e26c-4e2b-94fb-88b6cef0ca6e is duplicated. Please update it.